start-a-node.md

```diff
@@ -24,12 +24,14 @@ The `start` command supports the following flags, as well as [logging flags](coc
 Flag | Description
 -----|-----------
 `--attrs` | Node-level attributes specifying topography or machine capabilities. Topography might include datacenter designation (e.g., `us-west-1a`, `us-west-1b`, `us-east-1c`). Machine capabilities might include specialized hardware or number of cores (e.g., `gpu`, `x16c`). <br><br>Attributes can be arbitrary strings separated by colons. The relative geographic proximity of two nodes is inferred from the common prefix of the attributes list, so topographic attributes should be specified first and in the same order for all nodes, for example: <br><br> `--attrs=us-west-1b:gpu`
-`--certs` | The path to the directory containing the node's [security certificates](create-security-certificates.html). When starting the node with security (i.e., without the `--insecure` flag), the `--certs` flag is required. <br><br> **Default:** certs 
+`--cache` | The total size for caches, shared evenly if there are multiple storage devices. This can be in any bytes-based unit, for example: <br><br>`--cache=1000000000  -> 1000000000 bytes`<br>`--cache=1GB         -> 1000000000 bytes`<br>`--cache=1GiB        -> 0.93132257461548 bytes`
```

Comment:
The last example in this line is incorrect. 1 GiB is 1073741824 bytes. The second example is technically correct, but I thought we had decided to treat GB and GiB as synonymous (both meaning GiB). In that case I wouldn't show both examples.